src/components/articles/createArticles/createArticles.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import ReactQuill from 'react-quill';
+import { connect } from 'react-redux';
+import './createArticles.scss';
+import 'react-quill/dist/quill.bubble.css';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import Joi from 'joi-browser';
+import PropTypes from 'prop-types';
+import createArticlesAction from '../../../redux/actions/article';
+import { modules, formats } from './editor';
+
+export class CreateArticles extends React.Component {
+  schema = {
+    title: Joi.string()
+      .required()
+      .label('Title'),
+    description: Joi.string()
+      .required()
+      .label('Description'),
+    body: Joi.string()
+      .required()
+      .label('Article Body'),
+  };
+
+  constructor(props) {
+    super(props);
+    this.modules = modules;
+    this.formats = formats;
+    this.state = {
+      title: '',
+      body: '',
+      description: '',
+      schem: '',
+      images: [],
+      imageWidget: cloudinary.createUploadWidget(
+        {
+          cloudName: 'nshuti-jonathan',
+          uploadPreset: 'f31rkki4',
+          multiple: false,
+          cropping: true,
+          croppingShowBackButton: true,
+          singleUploadAutoClose: false,
+        },
+        (error, result) => {
+          if (!error && result && result.event === 'success') {
+            const { images } = this.state;
+            images[0] = result.info.secure_url;
+            images[0] = images[0].toString();
+            this.setState({ images });
+          }
+        },
+      ),
+    };
+    this.handleChangeField = this.handleChangeField.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentWillReceiveProps = (nextProps) => {
+    const { createdArticleData } = nextProps;
+
+    if (createdArticleData && Array.isArray(createdArticleData.err)) {
+      createdArticleData.err.forEach(({ message }) => {
+        toast.error(message);
+      });
+    }
+  };
+
+  handleChange = (html) => {
+    this.setState({ body: html });
+  };
+
+  showWidget = () => {
+    const { imageWidget } = this.state;
+    imageWidget.open();
+  };
+
+  handleChangeField(key, event) {
+    this.setState({
+      [key]: event.target.value,
+    });
+  }
+
+  handleSubmit() {
+    const { createArticlesAction: createArticle } = this.props;
+    createArticle(this.state, this.schema);
+  }
+
+  render() {
+    const { title, body, description } = this.state;
+    return (
+      <div className="col-12 col-lg-6 offset-lg-3">
+        <h3>Create Article</h3>
+        <input
```

Comment:
Hi use labels for inputs instead of just placeholders eg above the article title, add the article label like the way it was done in login and signup

Reply:
@SokoPaulSokool I have used labels and removed placeholders